cmd/experimental/kjobctl/apis/v1alpha1/volumebundle_types.go

```diff
@@ -39,6 +38,7 @@ type VolumeBundleSpec struct {
 	// envVars are environment variables that refer to absolute paths in the container filesystem.
 	// These key/value pairs will be available in containers as environment variables.
 	// +optional
+	// +kubebuilder:validation:MaxItems=5
```

Comment:
@trasc I hope it will enough. This is max that I can set.

Reply:
Yeah, working with arrays is expensive from the point of view of CEL. If we have an idea of the maximum of items that we're going to have is best to limit it.

Reply:
@IrvingMg thanks for investigation. Removed this constraint.

Reply:
I think here is most problem is regex.

Reply:
According to the docs, the longer the slice and the regex, the more expensive the rule cost. 

Here is more info: https://kubernetes.io/docs/reference/using-api/cel/#resource-constraints